task16.py

```diff
@@ -0,0 +1,17 @@
+# Требуется вычислить, сколько раз встречается некоторое число X в массиве A. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+# В последующих  строках записаны N целых чисел Ai
+# (можно использовать псевдогенерацию). Последняя строка содержит число X.
+
+
+import random
+n = int(input("Введите длинну строки "))
+lst = [random.randint(-2, 2) for i in range(n)]
+count = 0
+x = int(input("Введите искомое число"))
+
+for i in range(1, len(lst)):
```

Comment:
У вас уже есть параметр которых содержит длину списка.
```suggestion
for i in range(1, n):
```